src/components/App.js

```diff
@@ -2,64 +1,72 @@
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 
-import { color, notify } from '../utils'
-import Controls from './Controls'
-import Timer from './Timer'
+import { color, notify } from '../utils';
+import Controls from './Controls';
+import Timer from './Timer';
 
 const props = {
-  study: {duration: 1500},
-  rest: {duration: 300},
+	study: { duration: 1500 },
+	rest: { duration: 300 },
 };
 
 const App = () => {
-  const [time, setTime] = useState(1500);
-  const [isRunning, setRunning] = useState(false);
-  const [isStudying, setStudying] = useState()
+	const [time, setTime] = useState(1500);
+	const [isRunning, setRunning] = useState(false);
+	const [isStudying, setStudying] = useState();
 
-  useEffect(() => {  
-    const interval = setInterval(() => {
-      if (time === 0) {
-        end();
-        return () => clearInterval(interval);
-      }
-  
-      if(isRunning) {
-        setTime( time => time - 1);
-      }
-    }, 1000);
+	useEffect(() => {
+		const interval = setInterval(() => {
+			if (time === 0) {
+				end();
+				return () => clearInterval(interval);
+			}
 
-    return () => clearInterval(interval);
-  });
+			if (isRunning) {
+				setTime((time) => time - 1);
+			}
+		}, 1000);
 
-  const play = () => {
-    setRunning(!isRunning);
-  }
-  
+		return () => clearInterval(interval);
+	});
 
-  const end = (automated) => {
-    if(time !== 0 && !window.confirm("Are you sure you want to end?"))  { return; }
-    let duration = (isStudying ? props.study.duration : props.rest.duration);
-    setTime(duration);
-    setStudying(!isStudying);
-    setRunning(false);
-    color(isStudying, time);
-    notify('Session Finished');
-  }
+	const play = () => {
+		setRunning(!isRunning);
+	};
 
-  const restart = () => {
-    if(!window.confirm("Are you sure you want to restart?"))  { return; }
-    let duration = (isStudying ? props.rest.duration : props.study.duration);
-    setTime(duration);
-    setRunning(false);
-    notify('Session Restarted')
-    
-  }
+	const end = (automated) => {
+		if (time !== 0 && !window.confirm('Are you sure you want to end?')) {
+			return;
+		}
+		let duration = isStudying ? props.study.duration : props.rest.duration;
+		setTime(duration);
+		setStudying(!isStudying);
+		setRunning(false);
+		color(isStudying, time);
+		notify('Session Finished');
+	};
 
-  return (
-    <div className="Pomall">
-      <Timer time={time}></Timer>
-      <Controls isRunning={isRunning} restart={restart} play={play} end={end}></Controls>
-    </div>
-  )
+	const restart = () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/SamooMei/Pomall/pullRequest?prid=7386623)
```suggestion
		if (!window.confirm("Are you sure you want to restart?")) {
```